packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -132,6 +136,13 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
       const publishedPolicy = await PolicyServiceClient.readPolicy(policy_id, { withStatus: true })
       expect(publishedPolicy.status).toBe('pending')
       expect(publishedPolicy?.rules[0]?.states).toMatchObject({})
+
+      const { status, ...policyWithoutStatus } = publishedPolicy
+      const kafkaPolicy = mockStream.write.mock.calls[0][0]
```

Comment:
Could do a double array destructure here if you wanna be super fancy!
```suggestion
      const [[kafkaPolicy]] = mockStream.write.mock.calls
```